### DISCUSSION TITLE
Project Browser?

### DISCUSSION THREAD

**Discussion by danielbower** [coteditor/CotEditor#1411] [Q&A] [answered]
I'm pretty sure the answer to this is no, but there is I chance I missed a setting: wondering if CotEditor has a project/folder browser?

If not, does anyone know if this is a planned feature or one under consideration?

Cheers!

**Comment by 1024jp** [comment]
The short answer is "No" unfortunately.
As you expect, CotEditer doesn't have any project-related features, even it is a popular demand.
The original request is #278 and it still opens with no progress.

CotEditor is designed as a document-based (window per file) application the developer (namely, I) have less motivation to make it support projects although I don't disagree with it.

**Reply by danielbower** [reply]
Thanks for the reply and apologies for the dupe issue. I couldn't find the original for some reason!

Great work on Cot, it's still my go-to even without project support.

**Reply by ghost** [reply]
Set open file in new tab in the settings and use ctrl-o to open another file is one way. I do miss global search though.

**Comment by 1024jp** [comment, marked as answer]
I haven’t updated this thread, but the folder browser was now implemented in CotEditor 5.0.0, which was released in late 2024.
